components/proxy/src/main/java/com/hotels/styx/startup/AdminServerSetUp.java

```diff
@@ -32,6 +32,7 @@ public static HttpServer createAdminServer(StyxServerComponents config) {
         // TODO: Pass all backend Service Registries to AdminServerBuilder:
         // - only one backendServicesRegistry is passed in to the admin interface. Instead we
         //   should pass all of them:
+        // The comment seems to refer to part of a larger piece of work. Is there an issue to associate this with?
```

Comment:
Haha this is your own comment @kvosper: Styx server refactoring (https://github.com/HotelsDotCom/styx/pull/116).

If you no longer remember please remove.

Reply:
As the first line of the comment says, I only moved the comment here from `StyxServer`. I don't believe I wrote it originally. If it doesn't mean anything to you either, I will remove it.

Reply:
Ok. Please remove.